voicevox_engine/model.py

```diff
@@ -113,13 +113,34 @@ class SpeakerStyle(BaseModel):
     id: int = Field(title="スタイルID")
 
 
+class SpeakerSupportSynthesisMorphing(str, Enum):
+    PERMIT = "PERMIT"
+    SELF_MORPHING_ONLY = "SELF_MORPHING_ONLY"
+    PROHIBIT = "PROHIBIT"
```

Comment:
それぞれの想定をコメントしておくとわかりやすそうですね！！こうでしょうか。
```suggestion
    PERMIT = "PERMIT" # 全て許可
    SELF_MORPHING_ONLY = "SELF_MORPHING_ONLY" # 同じ話者内でのみ許可
    PROHIBIT = "PROHIBIT" # 全て禁止
```

あと些細ですが、なかなか見かけない用語だなと思いました。HTTP系だとALLOWとFORBID？
（たまに見る403 Forbidden）

Reply:
ここでの許可、禁止はライセンス上での提示くらいの感覚でいたので`permit`，`prohibit`ぐらい形式ばって強い語句の方が良いのかなと思い選びました．
（それはそれとして`PERMITED`の方が適切な気がしてきました）

Reply:
修正しました